.gitignore

```diff
@@ -1,3 +1,8 @@
+#: Secrets file
+fleetshare_secrets.py
+known_hosts
+.vscode/
```

Comment:
**suggestion**: we typically place project settings in `.vscode/settings.json`. It might be best to not ignore it and grab the defaults from https://github.com/agrc/python. The only thing about this that is annoying is the python path between developers is usually different. My flow for that has been to update it for my setup, and never commit my changes.

Reply:
Again, I think this is better as part of a larger discussion around standardization, not as a code review item when I wasn't aware of its existence.